chat-openai-sample/README.md

```diff
@@ -0,0 +1,52 @@
+
+## Prerequisites
+
+- Create an Azure account with an active subscription. For details, see [Create an account for free](https://azure.microsoft.com/free/).
+- Install [Visual Studio Code](https://code.visualstudio.com/download).
+- Install [Node.js](https://nodejs.org/en/download).
+- Create an Azure Communication Services resource. For details, see [Create an Azure Communication Resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource). You need to record your resource **connection string** for this sample.
+- An Azure OpenAI Resource and Deployed Model. See [instructions](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource?pivots=web-portal).
+
+## Before running the sample for the first time
+
+1. Open an instance of PowerShell, Windows Terminal, Command Prompt or equivalent and navigate to the directory where you want to clone the sample.
+2. git clone `https://github.com/Azure-Samples/communication-services-javascript-quickstarts.git`.
+3. cd into the `chat-openai-sample` folder.
+4. From the root of the `chat-openai-sample` folder run `npm install`.
+
+### Setup and host your Azure DevTunnel
+
+[Azure DevTunnels](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) is an Azure service that enables you to share local web services hosted on the internet. Use the following commands to connect your local development environment to the public internet. This creates a tunnel with a persistent endpoint URL and enables anonymous access. We use this endpoint to notify your application of chat events from the Azure Communication Services Chat service.
+
+```bash
+devtunnel create --allow-anonymous
+devtunnel port create -p 8080
+devtunnel host
+```
+
+### Configuring application
+
+Open the `.env` file to configure the following settings:
+
+1. `PORT`: Localhost port to run the server app on.
+1. `CONNECTION_STRING`: Azure Communication Services resource connection string.
+1. `ACS_URL_ENDPOINT`: Azure Communication Services resource URL endpoint.
+1. `AZURE_OPENAI_SERVICE_KEY`: Azure Open AI service key.
+1. `AZURE_OPENAI_SERVICE_ENDPOINT`: Azure Open AI endpoint.
+1. `AZURE_OPENAI_DEPLOYMENT_MODEL_NAME`: Azure Open AI deployment name.
+
+### Run app locally
+
+1. Open a new Powershell window, cd into the `chat-openai-sample` folder and run `npm run dev`.
+1. The browser displays the following dialog box. If not navigate to `http://localhost:8080/`.
+1. To test the AI analysis API endpoint on your local machine, in another new Powersehll window for the same directory, run `npm run client` to observe how messages are generated and processed.
+1. Register an EventGrid Webhook for the `ChatMessageReceived` event that points to your DevTunnel URI for `<DevTunnelUri>/api/chatMessageReceived`.
```

Comment:
For the Event Grid processing, it appears here as a step without any prior mention about it, so it might be confusing.

A couple of ideas:

1. Explicitly indicate it is part of the quickstart by describing it in the introduction and add prerequisite steps to set up Event Grid.
2. Have a separate optional section for all Event Grid related setup and what to look out for when the quickstart app is running.

Reply:
Second this.